deploy-spoke/csr_autoapprover.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+# Set path during systemd execution
+export PATH=/root/.local/bin:/root/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+
+# Wait for API to come online
+until [ $(curl -k -s https://api:6443/version?timeout=10s | jq -r '.major' | grep -v null | wc -l) -eq 1 ]; do
+    echo "Waiting for API..."
+    sleep 10
+done
+
+# Begin looking for and signing CSRs to activate nodes
+CLUSTER_DATA_FOLDER="/var/home/core/cluster_access_data"
+export KUBECONFIG=$(ls $CLUSTER_DATA_FOLDER/*/kubeconfig-* | head -1)
+
+if [ ! -f "${KUBECONFIG}" ]; then
+    echo "ERROR: Could not find kubeconfig file for cluster"
```

Comment:
Maybe it's better to:

```
echo "WARN: Could not find Kubeconfig file for cluster"
exit 0
```

Reason: 

- On day 2, the final client will need to reboot the cluster at some point, and He will see that error (maybe in a real error situation not related with this topic.

WDYT @flaper87

Reply:
done, disable unit and remove kubeconfig at the end of the script